### ISSUE TITLE
How to run queries?

### ISSUE THREAD

**Issue by mnpenner** [jimkyndemeyer/js-graphql-intellij-plugin#10]
First off, thanks for making this awesome plugin. It does way more than I would have expected! All I really wanted was syntax highlighting, but this is much better.

I got completions working with this minimalist `graphql.config.json` file (posting here for other n00bs):

```json
{
  "schema": {
    "request": {
      "url": "http://localhost:8080/graphql",
      "method": "POST",
      "postIntrospectionQuery": true
    }

  },

  "endpoints": [{
    "name": "MyProject",
    "url": "http://localhost:8080/graphql"
  }]
}
```

And I've created a .graphql scratch, but I can't figure out how to run my query. The `Query result` tab is blank, and there's no run button that I can see.

What am I missing?

![image](https://cloud.githubusercontent.com/assets/1112998/15370471/0df2b6bc-1cec-11e6-876a-ef799f6f57fc.png)

**Comment by jimkyndemeyer** [comment]
Hi Mark.

Thanks for using the plugin.

If you create aregular .graphql file the editor toolbar should appear as expected.

I typically have a query.graphql in the project root. You can have multiple queries in that single file. When you execute the query with the caret inside a query it's the only one to be executed. Referenced fragments are automatically included.

The shortcut to execute a query is Ctrl+Enter/Cmd+Enter depending on platform.

Let me know how it works out.

/Jim.

**Comment by mnpenner** [comment]
Aha! So it has to be a real file. Is there any way to get it to work with scratch files (Ctrl+Alt+Shift+Ins)? And can the result JSON be formatted? If not, no worries. They're just nice-to-haves.

**Comment by jimkyndemeyer** [comment]
Just released 1.3.0 with #11 

Let me know if there's any issues.

/Jim.

**Comment by mnpenner** [comment]
That's fantastic. Thank you !! Works perfectly.
